### PR TITLE
✨ Destination Teradata Vantage: Enabled dbt custom transformation for teradata destination connector

### PR DESCRIPTION
**Pull request by sc250072**
## What
Enabling dbt support to teradata destination connector

## How
Added supportsDbt: true to metadata.yaml and spec.json of teradata destination connector to enable dbt customization/transformation for teradata.
## Recommended reading order
1. `metadata.yaml`
2. `spec.json`
3. `teradata.md`

## 🚨 User Impact 🚨
There are no breaking changes. 
Users can use teradata dbt custom transformation with teradata airbyte destination connector.

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added
- 
image

image

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

**Review by evantahler — APPROVED** [review]

**Review by evantahler — CHANGES_REQUESTED** [review]
Please increment the connector's version in metadata.yaml, and add a changelog entry in the docs

**Comment by evantahler** [comment]
Please also complete the above chceklist

**Review by evantahler — APPROVED** [review]
@marcosmarxm I think this looks good! I'm assigning the final review & merge to you

File: airbyte-integrations/connectors/destination-teradata/metadata.yaml [hunk 1]
```diff
@@ -2,19 +2,24 @@ data:
...
+  dockerImageTag: 0.1.3
   dockerRepository: airbyte/destination-teradata
   githubIssueLabel: destination-teradata
   icon: teradata.svg
   license: MIT
   name: Teradata Vantage
+  normalizationConfig:
+    normalizationIntegrationType: null
+    normalizationRepository: null
+    normalizationTag: null
```
Comment (evantahler) [inline airbyte-integrations/connectors/destination-teradata/metadata.yaml:14]: I would remove this.  I think your testing env was problematic

Comment (SatishChGit) [reply airbyte-integrations/connectors/destination-teradata/metadata.yaml:14]: I tried to setup airbyte on multiple instances but observed same behavior. I updated actor_definition_version supportsDBT as true and can able to see custom transformation UI only. normalizationConfig fields are kept empty. So with this I am assuming normalizationConfig is not required to specified with null

Comment (SatishChGit) [reply airbyte-integrations/connectors/destination-teradata/metadata.yaml:14]: Shell I remove these and recommit the file?

Comment (evantahler) [reply airbyte-integrations/connectors/destination-teradata/metadata.yaml:14]: please remove them

**Comment by evantahler** [comment]
Linking https://github.com/airbytehq/airbyte/issues/29872 which was likely confounding the research here

**Comment by marcosmarxm** [comment]
Sorry for the long delay in getting this reviewed and merged. Our team is currently finishing the August Hackathon review contributions over the next two weeks, and after that, we will return to the community backlog.

**Comment by evantahler** [comment]
Ping! I think this should be good to go.

**Comment by evantahler** [comment]
Closing in favor of https://github.com/airbytehq/airbyte/pull/30740

**Pull request closed without merging**